backend/src/services/reservationService.ts

```diff
@@ -8,12 +8,16 @@ const numConcurrentReservations: number = 1;
 const getReservationFromRange = async (startTime: Date, endTime: Date) => {
```

Comment:
Tarvitseeko olla kaksi varaukset aikavälillä hakevaa funktiota? Miksi getInTimeRange ei ole riittävä?

Reply:
Varmasti siis on hyvä syy mut jäin miettiii että mikä se on